CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ if(MSVC)
 else()
 	check_cxx_compiler_flag("-Wall -Werror" CXX_SUPPORTS_WERROR)
 	if(CXX_SUPPORTS_WERROR)
-		set(DPSIM_CXX_FLAGS -Wall -Werror)
```

Comment:
Yes, although it makes the compilation with clang fail. However, because of an error in the VillasNode headers, clang won't compile dpsim anyway, so it currently does make no difference.